### PR TITLE
Sofia/error handling

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/src/lib.rs [hunk 1]
```diff
@@ -38,17 +38,31 @@ pub enum CreationError {
...
-            CreationError::Negative => "Negative",
-            CreationError::Zero => "Zero",
-        }
-    }
-}
\ No newline at end of file
+impl error::Error for CreationError {}
+
+
+//impl fmt::Display for CreationError {
```
Comment (Geravant) [inline Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/src/lib.rs:55]: Do we need the commented part here?

File: Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/task-info.yaml [hunk 2]
```diff
@@ -5,16 +5,16 @@ files:
 - name: src/lib.rs
   visible: true
   placeholders:
-  - offset: 187
-    length: 17
+  - offset: 191
+    length: 21
     placeholder_text: ???
```
Comment (Geravant) [inline Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/task-info.yaml:10]: we can add the /**/ around the placeholder text

File: Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/task.md [hunk 3]
```diff
@@ -13,29 +13,43 @@ How can we lump these errors into one general error?
...
 <div class="hint">
-  `Result`s must be "used", that is, you'll get a warning if you don't handle a `Result` that you get in your function.
+<code>Result</code>s must be "used", that is, you'll
+get a warning if you don't handle a <code>Result</code> that you get in your
+function. 
 
-  Read more about that in the `std::result` module docs:
-  https://doc.rust-lang.org/std/result/#results-must-be-used
+Read more about that in the <code>std::result</code> module docs:
+<a>https://doc.rust-lang.org/std/result/#results-must-be-used</a>
```
Comment (Geravant) [inline Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/task.md:54]: The closing tag is visible at the moment

**Review by Geravant — CHANGES_REQUESTED** [review]
Great job, just one question about the Multiple Errors task and minor text stuff

File: Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/src/lib.rs (hunk 1 above)
Comment (sofiiako) [reply Recoverable and Unrecoverable Errors/Error Handling/Multiple Errors/src/lib.rs:55]: oops)

**Comment by sofiiako** [comment]
@Geravant corrected

File: Common Programming Concepts/section-info.yaml [hunk 4]
```diff
@@ -3,4 +3,3 @@ content:
 - Data Types
 - Functions
 - If
-- Variables, Functions and If Test
```
Comment (Geravant) [inline Common Programming Concepts/section-info.yaml:6]: This is from another lesson and named obsolete

**Review by Geravant — APPROVED** [review]

**Pull request merged**